.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: node_js
 node_js:
-  - 8
-  - 9
   - 10
+  - 12
+  - 14
+env:
+- AGENDA_VERSION=^4.0.0
+- AGENDA_VERSION=^3.1.0
 matrix:
   fast_finish: true
+# Run npm test, unless it's Node.js 10 and then run just lint+Express tests
+# since Hapi isn't compatible with Node.js 10
+script: 'if [ "${TRAVIS_NODE_VERSION:0:2}" == "10" ]; then npm run lint && npm run ava test/test-express.js; else npm test; fi'
```

Comment:
So I added a conditional here to skip Hapi tests on Node.js 10 — [which isn't planned](https://github.com/nodejs/Release#release-schedule) to stay around very long anyway:

![image](https://user-images.githubusercontent.com/87168/104967664-93b1f780-59ec-11eb-84b7-e56b1cc25323.png)

I'd expect Hapi/Express restructuring need some changes here anyway later.

This is a lot of testing btw; probably no need to test couple Agenda versions on every Node.js version? We can simplify later.